### ISSUE TITLE
Updating order status to shipped changes the shipping adjustment to "included"

### ISSUE THREAD

**Issue by moldedjelly** [craftcms/commerce#1243]
### Description
Using the control panel, updating an order status to "shipped" sets the shipping adjustment (a flat-rate shipping cost) to "included" when it shouldn't be and as a result reduces the Total Price of the order by the shipping cost and sets the order as overpaid.

### Additional info
- Craft version: 3.4.1
- PHP version: 7.2.12

**Comment by lukeholder** [comment]
Where is this happening, on the order edit page? Is the order marked as complete already? Need some more info.

**Comment by moldedjelly** [comment]
@lukeholder I'm not sure if my tax rule of having the tax "included in line item total + total discounts + total shipping" is maybe getting it confused? 
[screenshot of tax settings](https://www.inkahoots.com.au/screenshots/uqp/tax_settings.png)

**Comment by matthisamoto** [comment]
I am seeing this behavior as well.

This occurs when orders are changed from Processing to Shipped status, it appears to count the shipping and tax charges as included only once this status change happens. We do not have the tax rule set to be included, it is separated. 

We recently did a Craft 2 => 3 update within the last month, which took us to Commerce 2, then did the Commerce 3 update the other week when it was released.

What else can I provide to help look into this?

![Screen-Shot-2020-02-07-at-10 21 07-AM](https://user-images.githubusercontent.com/547172/74046333-f81ea700-4993-11ea-8f21-5ef893af7cb4.png)

###Application Info
PHP version	7.3.14
OS version	Linux 3.10.0-957.27.2.el7.x86_64
Database driver & version	MySQL 5.7.29
Image driver & version	GD 7.3.14
Craft edition & version	Craft Pro 3.4.3
Yii version	2.0.32
Twig version	2.12.3
Guzzle version	6.5.1
Imagine version	1.2.3-dev

###Plugins
Calendar	3.0.0
Cookies	1.1.11
Craft Commerce	3.0.5
Feed Me	4.2.0.1
Field Manager	2.1.1
Imager	v2.3.1
Many to Many Field Type	1.0.2.2
Navigation	1.3.2
Redactor	2.5.0
Retour	3.1.33
Smart Map	3.2.2.1
Stripe for Craft Commerce	2.2.3
SuperSort	3.0.1
Typogrify	1.1.18

**Comment by matthisamoto** [comment]
The Order Details page appears to be incorrectly displaying the Shipping adjustment total as the per item rate ($2) as well, the Ground shipping total minimum cost is $4, which coupled with the $1.65 Tax adjustment accounts for the discrepancy in displayed Total Price at $20 vs Paid Amount at $25.65.

![Screen Shot 2020-02-07 at 10 35 05 AM](https://user-images.githubusercontent.com/547172/74047210-97906980-4995-11ea-9a17-b88646a4b43d.png)

**Comment by lukeholder** [comment]
Looking into this now. Stay tuned.

**Comment by lukeholder** [comment]
Fixed the adjustment checkbox issue for the next release of Commerce.

To get the fix early, change your `craftcms/commerce` requirement in composer.json to:

```json
"require": {
  "craftcms/commerce": "dev-develop#b9974fbd92e5feaa129e8166b6ede74bc174913b as 3.0.6",
  "...": "..."
}
```

Then run `composer update`.

**Comment by lukeholder** [comment]
@matthisamoto Please open another ticket for an unreleased issue.
